Packs/Kafka/Integrations/KafkaV3/KafkaV3.py

```diff
@@ -0,0 +1,633 @@
+    if demisto_params.get('isFetch', False):
+        valid_fetch = check_params(kafka=kafka,
+                                   topic=demisto_params.get('topic', None),
+                                   partitions=handle_empty(argToList(demisto_params.get('partition', None)), None),
+                                   offset=handle_empty(demisto_params.get('offset', 'earliest'), 'earliest'))
+    if connection_test != 'ok':
+        return connection_test
+    elif not valid_fetch:
+        return 'Failed'
+    else:
+        return 'ok'
+
+
+def print_topics(kafka: KafkaCommunicator, demisto_args: dict) -> Union[CommandResults, str]:
+    """Print available topics in Broker
+
+    Args:
+        kafka: initialized KafkaCommunicator object to preform actions with.
+        demisto_args: The demisto command arguments.
+
+    Return CommandResults withe the detailed topics, 'No topics found.' if no topics were found.
+    """
+    include_offsets = demisto_args.get('include_offsets', 'true') == 'true'
+    kafka_topics = kafka.get_topics().values()
+    if kafka_topics:
+        topics = []
+        for topic in kafka_topics:
+            partitions = []
+            for partition in topic.partitions.values():
+                partition_output = {'ID': partition.id}
+                if include_offsets:
+                    try:
+                        partition_output['EarliestOffset'], partition_output['OldestOffset'] = kafka.get_partition_offsets(
+                            topic=topic.topic, partition=partition.id)
+                    except KafkaException as e:
+                        if 'Unknown partition' not in str(e):
+                            raise e
+                partitions.append(partition_output)
+
+            topics.append({
+                'Name': topic.topic,
+                'Partitions': partitions
+            })
+
+        readable_output = tableToMarkdown('Kafka Topics', topics)
+
+        return CommandResults(
+            outputs_prefix='Kafka.Topic',
+            outputs_key_field='Name',
+            outputs=topics,
+            readable_output=readable_output,
+        )
+
+    else:
+        return 'No topics found.'
+
+
+def produce_message(kafka: KafkaCommunicator, demisto_args: dict) -> None:
+    """Producing message to kafka topic
+
+    Args:
+        kafka: initialized KafkaCommunicator object to preform actions with.
+        demisto_args: The demisto command arguments.
+
+    kafka.delivery_report is called when producing is done and it returns the relevant results.
+    """
+    topic = demisto_args.get('topic')
+    value = demisto_args.get('value')
+    partition = demisto_args.get('partition')
+
+    partition = str(partition)
+    if partition.isdigit():
+        partition = int(partition)  # type: ignore
+    else:
+        partition = None  # type: ignore
+
+    kafka.produce(
+        value=str(value),
+        topic=str(topic),
+        partition=partition  # type: ignore
+    )
+
```

Comment:
lets return here some output that the message was produced successfully

Reply:
It is returned in the callback function